x-pack/test_serverless/api_integration/test_suites/observability/apm_api_integration/service_maps/service_maps.ts

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ApmSynthtraceEsClient } from '@kbn/apm-synthtrace';
+import expect from 'expect';
+import { serviceMap, timerange } from '@kbn/apm-synthtrace-client';
+import { Readable } from 'stream';
+import type { InternalRequestHeader, RoleCredentials } from '../../../../../shared/services';
+import { APMFtrContextProvider } from '../common/services';
+
+export default function ({ getService }: APMFtrContextProvider) {
+  const apmApiClient = getService('apmApiClient');
+  const svlUserManager = getService('svlUserManager');
+  const svlCommonApi = getService('svlCommonApi');
+  const synthtrace = getService('synthtrace');
+
+  const start = new Date('2024-06-01T00:00:00.000Z').getTime();
+  const end = new Date('2024-06-01T00:05:00.000Z').getTime() - 1;
```

Comment:
Nit - I think it will be easier to read if we have the exact date and remove the `-1`, wdyt?

Reply:
Yeah, this was copied form another test. good catch